### ISSUE TITLE
Re-introduce a way to change the root folder

### ISSUE THREAD

**Issue by tanriol** [ancestorak/brief#156]
This is frequently requested. I didn't like the old UI of this feature, but maybe we can introduce a better UI.

**Comment by tanriol** [comment]
See #155

**Comment by nohamelin** [comment]
The discussion in #114 is worth to read too.

But, first, please restore the old UI, and *when* a better implementation is conceived, then replace it. It don't need be so exposed in the Options window as before: a button for a sub-dialog would be enough, in the same way that it's done with the "Customize Brief via CSS" option.

**Comment by akwala** [comment]
From the [FAQ](https://github.com/ancestorak/brief/wiki/FAQ#brief-does-not-use-the-feeds-folder-synced-from-firefox-sync):

> If you want a better integration, you need to find the ID for your synchronized folder in the new Firefox install ... go to `about:config`, find `extensions.brief.homeFolder` and set it to the number you found.

Brief 2.5.5, in Firefox 58.0.2, does not consistently show the feeds even after setting `extensions.brief.homeFolder` as explained above. Although the feeds did appear after I first set the value of this preference, they stopped appearing at some point. The ID of the _Subscribed Feeds_ folder had not changed since I set the preference. The feeds reappeared when I added a new feed using Brief, but the newly added feed did not get added to the _Subscribed Feeds_ folder; I had to add it there as a Live Bookmark manually. Therefore, this workaround doesn't seem to result in an integration with Firefox Sync.
